wandb-scim/examples/teams_example.py

```diff
@@ -0,0 +1,109 @@
+# calling_module.py
+
+import requests
+import sys
+sys.path.append('../')
+from teams import Teams  # Assuming the Teams class is defined in teams.py
+
+def create_team(teams, display_name, member_id):
+    """
+    Creates a new team.
+
+    Args:
+        teams (Teams): An instance of the Teams class.
+        display_name (str): The display name of the new team.
+        member_id (str): The ID of the member to be added to the team.
+    """
+    try:
+        # Create a new team
+        create_team_response = teams.create(
+            request_payload={
+                "displayName": display_name,
+                "member": member_id
+            }
+        )
+        print(create_team_response)
+    except requests.exceptions.RequestException as e:
+        print(f"Error occurred during API request: {str(e)}")
+
+def get_team(teams, team_id):
+    """
+    Retrieves details of a specific team.
+
+    Args:
+        teams (Teams): An instance of the Teams class.
+        team_id (str): The ID of the team to retrieve.
+    """
+    try:
+        # Get team details
+        get_team_response = teams.get(team_id)
+        print(get_team_response)
+    except requests.exceptions.RequestException as e:
+        print(f"Error occurred during API request: {str(e)}")
+
+def get_all_teams(teams):
+    """
+    Retrieves details of all teams in the organization.
+
+    Args:
+        teams (Teams): An instance of the Teams class.
+    """
+    try:
+        # Get all teams in the organization
+        get_all_teams_response = teams.get_all()
+        print(get_all_teams_response)
+    except requests.exceptions.RequestException as e:
+        print(f"Error occurred during API request: {str(e)}")
+
+def update_team_add_member(teams, team_id, member_id):
+    """
+    Updates a team by adding a member.
+
+    Args:
+        teams (Teams): An instance of the Teams class.
+        team_id (str): The ID of the team to update.
+        member_id (str): The ID of the member to add to the team.
+    """
+    try:
+        # Update team by adding a member
+        update_team_response = teams.update(
+            team_id,
+            request_payload={"value": member_id}
+        )
+        print(update_team_response)
+    except requests.exceptions.RequestException as e:
+        print(f"Error occurred during API request: {str(e)}")
+
+def update_team_remove_member(teams, team_id, member_id):
+    """
+    Updates a team by removing a member.
+
+    Args:
+        teams (Teams): An instance of the Teams class.
+        team_id (str): The ID of the team to update.
+        member_id (str): The ID of the member to remove from the team.
```

Comment:
This needs to support removing multiple members in a single request, like the add operation